nss-day-cohort-39/tabloid-mvc-tears-for-fears
# tag issues

### Changes
-Added PostTagRepository, PostTagController, ManageAddTagsViewModel and ManageRemoveTagsModel

### Testing
- Navigate to My posts
- open details of post
- navigate to add tag page by clicking second pencil icon from the left
- add tag
- After redirected to post details navigate back to add tags and confirm that the same tag you just added is no longer an option to add
- go back to post details and navigate to remove tags by clicking the second trash can from the left and confirm the tag that you added is in this list and remove it from post.
